weaviate/data/replication/replication.py

```diff
@@ -22,3 +23,30 @@ class ConsistencyLevel(str, BaseEnum):
     ALL = auto()
     ONE = auto()
     QUORUM = auto()
+
+
+def name_consistency_level(level: Union[str, ConsistencyLevel]) -> str:
```

Comment:
This is actually not needed, if you implement `def __str__` for the `BaseEnum` class:

```python
class BaseEnum(Enum, metaclass=MetaEnum):
    def __str__(self):
        return self.name
```

This will let you do the following:
```python
>>> str(ConsistencyLevel.ALL)
'ALL'
```

etc

Reply:
What's about the validation?
The caller relays on this function to raise ValueError in case of invalid level
 ```
if level not in ConsistencyLevel:
        raise ValueError(f"invalid ConsistencyLevel: {level}") #here
    if isinstance(level, ConsistencyLevel):
        return level.name
    else:
        return level
```

Reply:
Another alternative is to simply refactor the `ConsistencyLevel` class:

```python
class ConsistencyLevel(str, BaseEnum):
    ALL = 'ALL'
    ONE = 'ONE'
    QUORUM = 'QUORUM'
```

This allows us to use the typed enum, but they also behave like strings:
```python
>>> ConsistencyLevel.ALL == 'ALL'
True
```